build-system/tasks/visual-diff.js

```diff
@@ -116,7 +116,7 @@ function setPercyBranch() {
 async function launchWebServer() {
   const gulpServeAsync = execScriptAsync(
       `gulp serve --host ${HOST} --port ${PORT} ${process.env.WEBSERVER_QUIET}`,
-      {stdio: 'inherit'});
+      getStdioOptions(argv.gulp_debug || argv.webserver_debug));
```

Comment:
I think the server output should depend only on whether `webserver_debug` is enabled. This could be simplified to:
```
{'stdio': argv.webserver_debug ? 'inherit' : 'ignore'},
```

Reply:
Sure, but in this case argv.gulp_debug becomes useless. I'll remove it too :)